helm-chart/templates/_helpers.tpl

```diff
@@ -141,3 +139,33 @@ The name of the service account to use for backend pods
   value: "{{ .Release.Namespace }}"
 {{- end }}
 {{- end }}
+
+{{- define "cvat.sharedClickhouseEnv" }}
+{{- if .Values.analytics.enabled}}
```

Comment:
So how do you actually switch to an external instance? In `Chart.yml`, the Clickhouse subchart is enabled if `analytics.enabled` is true, so there's no way to disable the internal instance without disabling analytics altogether.